scripts/pylib/twister/twisterlib/testsuite.py

```diff
@@ -407,13 +407,15 @@ def add_testcase(self, name, freeform=False):
     def get_unique(testsuite_root, workdir, name):
 
         canonical_testsuite_root = os.path.realpath(testsuite_root)
-        if Path(canonical_zephyr_base) in Path(canonical_testsuite_root).parents:
-            # This is in ZEPHYR_BASE, so include path in name for uniqueness
-            # FIXME: We should not depend on path of test for unique names.
-            relative_ts_root = os.path.relpath(canonical_testsuite_root,
+        relative_ts_root = os.path.relpath(canonical_testsuite_root,
                                                start=canonical_zephyr_base)
-        else:
-            relative_ts_root = ""
+        # This is in ZEPHYR_BASE, so include path in name for uniqueness
+        # FIXME: We should not depend on path of test for unique names.
+        if Path(canonical_zephyr_base) not in Path(canonical_testsuite_root).parents:
```

Comment:
if you embed it in `is_outside_zephyr_base` code is self descriptive

Reply:
Done, also reworded the comment to better reflect the workflow.